src/main/java/org/wikivoyage/listings/language/english/English.java

```diff
@@ -51,18 +51,19 @@ public String getNameElement() {
      * Convert listing template into a WikivoyagePOI object.
      */
     @Override
-    public Listing parseListingTemplate(String article, TemplateNode template) {
+    public Listing parseListingTemplate(String article, TemplateNode template, String poiType) {
         // Type
-        String poiType;
-        if (template.getNameLowercase().equals("listing")) {
-            if (template.hasArgument("type")) {
-                poiType = template.getArgument("type");
-            } else {
-                poiType = "other";
-            }
-        } else {
-            poiType = template.getNameLowercase();
-        }
+    		if (poiType=="") {
```

Comment:
See https://stackoverflow.com/a/531825/226958 (even though that does not matter anymore if you switch to `null` instead of `""`)

Reply:
Sorry for this error, sometimes I get lost between different languages!